.storybook/preview.js

```diff
@@ -12,8 +13,10 @@ export const parameters = {
 export const decorators =[
   Story => (
     <>
-      <GlobalStyle />
-      <Story />
+      <MemoryRouter initialEntries={['/']}>
```

Comment:
Cool, didn't know that for storybook it is required to use `MemoryRouter`. I'll keep that in mind for my user story today 🥳